website/content/docs/concepts/namespace-api-lock.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Namespace API Lock
+description: Lock the Vault API on a per-namespace basis.
+---
+
+# Lock/Unlock
+
+Vault makes the API available (unlocked) by default for all namespaces. In
+this state, Vault can respond to all API/CLI ('API' from here on out) requests
+as it usually would.
+
+A Vault Administrator can lock the API for particular namespaces. In this state,
+Vault will block all but a select few API endpoints from responding to clients
+operating in a locked namespace (or a descendant of a locked namespace). The
+endpoints that do respond, the exempt paths, are largely the same as the Vault
+unauthenticated paths. They are mainly used for checking the status of various
+systems (e.g. `sys/health`), or for unlocking the API.
+
+When the API is locked for a particular namespace, it is also locked for all
+descendants of that namespace. If the API was already locked for a descendant
+that lock will remain, but Vault must be unlocked for the parent before
+unlocking the descendant.
+
+## Why?
+
+Blocking access to much of Vault can be an important break-glass tool in the
+event of unexpected behavior.
+
+For HCP Vault, this provides functionality analogous to sealing Vault, without
+the Vault Administrator requesting that the Managed Service Provider seal/unseal
+Vault.
+
+This feature also becomes valuable for secure multitenancy in a variety of Vault
+deployment strategies. One can restrict Vault access for just part of an
+organization, without affecting adjacent parts of the business. If unexpected
+behavior is detected in sub-organization A, an Administrator can disable Vault
+access for any entity under A, without disabling access for sub-organization B.
+Once the cause has been identified and resolved, the API can be unlocked for A.
+
+## Locking
+
+The API can be locked by running `vault namespace lock` (or via the API) while
+operating in the namespace to lock. Optionally, a subpath can be provided to
+lock a descendant of the current namespace.
+
+An unlock key will be returned, which can be used to unlock the API for that
+namespace. Preserve this key to unlock the API in the future.
+
+When the API is locked for a namespace, it will also be locked for all
+descendants of that namespace. If an authenticated client attempts to access
+Vault from a locked namespace, the returned error will inform that client of the
+locking namespace.
```

Comment:
Suggested text:

If an authenticated client attempts to access Vault from a locked namespace, the return error informs the client that they are attempting to access a locked namespace.